cluster-up/hack/common.sh

```diff
@@ -17,7 +17,7 @@ fi
 
 
 KUBEVIRTCI_CLUSTER_PATH=${KUBEVIRTCI_CLUSTER_PATH:-${KUBEVIRTCI_PATH}/cluster}
-KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-k8s-1.18}
+KUBEVIRT_PROVIDER=${KUBEVIRT_PROVIDER:-k8s-1.15}
```

Comment:
This should be 1.18

Reply:
`make generate` produced this so it looks like kubevirtci should be updated

Reply:
oh wait

Reply:
Or `make generate` updates this

Reply:
Hah should teach me to review late at night, didn't even notice it was in cluster-up. Yeah this is all generated so out of your control.